File: ProductsResourceServer/Model/Product/Product.cs
namespace ProductsResourceServer.Model.Product
{
    public class Product
    {
        public long Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public byte[] Image { get; set; }
    }
}

File: AngularCliCodeFlowPkce/src/app/Models/Product.ts
export class Product {
  id: number;
  name: string;
  description: string;
  image:any;

  constructor(id: number, name: string, description: string, image:any) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.image = image;
  }
}
